instructions/fault-injection/fault-injection.adoc

```diff
@@ -0,0 +1,150 @@
+= Chaos Testing
+include::_attributes.adoc[]
+
+Apply some chaos engineering by throwing in some HTTP errors or network delays. Understanding failure scenarios is a critical aspect of microservices architecture (aka distributed computing)
+
+[IMPORTANT]
+.Before Start
+====
+You should have only the following virtualservices and destinationrules in the `istio-tutorial` namespace
+
+[source, bash,subs="+macros,+attributes"]
+----
+$ oc -n istio-tutorial get destinationrule
+No resources found.
+
+$ oc -n istio-tutorial get virtualservice
+NAME       GATEWAYS             HOSTS   AGE
+customer   [customer-gateway]   [*]     18h
+----
+
+====
+
+[#503error]
+== HTTP Error 503
+
+By default, recommendation v1, v2 and v3 are being randomly load-balanced as that is the default behavior in Kubernetes/OpenShift
```

Comment:
This is the default behaviour in Istio, it's not kubernetes handling the load balancing but rather envoy directly accessing the endpoints

Reply:
OK, I'll replace with Istio. This was taken from the upstream tutorial.

Reply:
The upstream tutorial is wrong in that case :)